Sources/TSCBasic/ProcessEnv.swift

```diff
@@ -62,6 +62,15 @@ public enum ProcessEnv {
         return localFileSystem.currentWorkingDirectory
     }
 
+    public static var pathString: String {
+#if os(Windows)
+        let pathArg = "Path"
+#else
+        let pathArg = "PATH"
+#endif
+        return vars[pathArg] ?? ""
+    }
```

Comment:
Wouldn't you want to make this optional? It would then be more similar to the current behavior and `getEnvSearchPaths` already takes its `pathString` argument as an optional.

```suggestion
    public static var pathString: String? {
#if os(Windows)
        let pathArg = "Path"
#else
        let pathArg = "PATH"
#endif
        return vars[pathArg]
    }
```

Reply:
Are there any other places in code where `pathString` is passed an optional value? If not, maybe it could be made non-optional in the `getEnvSearchPaths` function signature?

Reply:
Yeap, there's a usage of `getEnvSearchPaths` in SwiftPM, and one more in SwiftDriver.
I've looked at the SwiftDriver usage briefly (it would require adding `?? ""`), and I think I agree with your initial suggestion now 👍.

(The usage in SwiftPM needs a similar fix to work properly on Windows, I'll send one more PR to fix it as well, if this one is merged)